gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineFormat.java

```diff
@@ -65,7 +66,7 @@ public void apply(Project project) {
                 Task spotlessJava = project.getTasks().getByName("spotlessJava");
                 Task spotlessApply = project.getTasks().getByName("spotlessApply");
                 if (eclipseFormattingEnabled(project) && !Files.exists(eclipseXml)) {
-                    spotlessJava.dependsOn(project.getTasks().findByPath(":baselineUpdateConfig"));
+                    spotlessJava.dependsOn(":baselineUpdateConfig");
```

Comment:
this is just style - no need to resolve the actual task ourselves since dependsOn takes a string